app/commands/codemod/writer.go

```diff
@@ -1,6 +1,7 @@
 package codemod
```

Comment:
should have a package comment, unless it's in another file for this package

Reply:
/ignore

Reply:
/ignore remove